cpp_api/api.cpp

```diff
@@ -0,0 +1,36 @@
+#include <ansi-c/ansi_c_language.h>
+
+#include <goto-programs/goto_model.h>
+#include <goto-programs/initialize_goto_model.h>
+
+#include <langapi/mode.h>
+
+#include <util/cmdline.h>
+#include <util/message.h>
+#include <util/options.h>
+
+#include "api.h"
+
+api_depst api_deps;
```

Comment:
I hope these are temporary. We definitely don't want to have globals. Also see below wrt sessions.